cleanlab/datalab/internal/issue_manager/outlier.py

```diff
@@ -103,7 +103,11 @@ def find_issues(
             t = cast(int, self.ood.params["t"])
             distances = knn_graph.data.reshape(-1, k)
             assert isinstance(distances, np.ndarray)
-            scores = transform_distances_to_scores(distances, k=k, t=t)
+            avg_distances = distances.mean(axis=1)
+            median_distance = np.median(avg_distances)
```

Comment:
[nit] This would be `median_avg_distance`

Reply:
I also think the current variable name is confusing and prefer `median_avg_distance`